client-java/controller/src/test/java/com/thrift/example/real/facebook/fb303/FacebookService.java

```diff
@@ -0,0 +1,9791 @@
+/**
```

Comment:
this is nearly 10 000 lines of code, which was automatically generated. fine.
does Thrift require a special tool to generate this code? or could it be called via a Maven plugin?

Reply:
ok, thx